bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/CachedVoiceRSSCloudImpl.java

```diff
@@ -75,12 +77,8 @@ public File getTextToSpeechAsFile(String apiKey, String text, String locale, Str
             writeText(txtFileInCache, text);
             // return from cache
             return audioFileInCache;
-        } catch (FileNotFoundException ex) {
-            logger.warn("Could not write {} to cache", audioFileInCache, ex);
-            return null;
         } catch (IOException ex) {
-            logger.error("Could not write {} to cache", audioFileInCache, ex);
-            return null;
+            throw new IOException("Could not write to cache file: " + ex.getMessage(), ex);
```

Comment:
Just checking: `IOException` is catched and handled somewhere, right? Previously it was not thrown from this method.

Reply:
Yes, it is handled by the caller.
It was already thrown by this method, but not at this place (elsewhere in the method).

Reply:
What I changed here is that the method will no more return null but will rather throw an exception.